Library/Formula/aws-sdk-cpp.rb

```diff
@@ -0,0 +1,70 @@
+class AwsSdkCpp < Formula
+  desc "AWS SDK for C++"
+  homepage "https://github.com/aws/aws-sdk-cpp"
+  url "https://github.com/aws/aws-sdk-cpp/archive/0.9.6.tar.gz"
+  sha256 "cf2dad62e50c5c5cff88a4ad959af3aa402aa7e8aa1692bdf807052c0fe4aeb9"
+  head "https://github.com/aws/aws-sdk-cpp.git"
+
+  ALL_CLIENTS = [
```

Comment:
@dunn is there any way to add a flag that accepts a string argument? If so we can easily just allow the user to specificy `--with-clients='s3;ec2'`.

In my use case, I'd like users to be able to just install `kinesis` and `firehose`, which definitely seems like a non-standard set of clients. It would be unfortunate if we ended up having to do the full build just to get these 2 (+ the core) that we want.